packages/preview/bytefield/0.0.1/typst.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name="bytefield"
+version = "0.0.1"
+entrypoint = "bytefield.typ"
+authors = ["Jomaway <https://github.com/jomaway>"]
+license = "MIT"
```

Comment:
could you `exclude` the `ipv4-example.png` file to reduce the package size? there are a bunch of other packages which exclude assets, you can take a look there.

Reply:
Sure, I will do after my vacation.

Reply:
@laurmaedje removed the `png` as requested.